webapp/fmprod/src/test/java/org/apache/oodt/cas/product/jaxrs/resources/DatasetResourceTest.java

```diff
@@ -70,7 +71,7 @@ public void testXmlMarshalling() throws IOException, JAXBException,
     List<Reference> references1 = new ArrayList<Reference>();
     references1.add(reference1);
 
-    Hashtable metadataEntries1 = new Hashtable<String, Object>();
+    HashMap<String, Object> metadataEntries1 = new HashMap<>();
```

Comment:
For clarification: @NGimhana Do you remember why this change was made? If this change was intentional, I think we can remove the import `HashTable` under imports.

Reply:
```suggestion
 @Deprecated
 public void addMetadata(Hashtable<String, Object> metadata) {
     addMetadata(metadata, false);
 }
```
Above method in [cas/Metadata.java](https://github.com/apache/oodt/blob/master/metadata/src/main/java/org/apache/oodt/cas/metadata/Metadata.java) is deprecated. Just affect to Test cases in this case. So we can remove Hashtable import.

Reply:
```java
@Deprecated
public void addMetadata(Hashtable<String, Object> metadata) {
    addMetadata(metadata, false);
}
```

Above method in [cas/Metadata.java](https://github.com/apache/oodt/blob/master/metadata/src/main/java/org/apache/oodt/cas/metadata/Metadata.java) is deprecated. Just affect to Test cases in this case. So we can remove Hashtable import.